frontend/src/components/Content.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { Wrapper } from './GlobalStyles';
+import { API_URL } from './config';
+import Loader from './Loader';
+
+const Content = () => {
+  // Local state variables for storing data from API
+  const [mostViews, setMostViews] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [accessToken] = useState(localStorage.getItem('accessToken'));
```

Comment:
Great to use localStorage to no be logged out when refreshing page. We will steal this ;)